src/rust/kj/random.rs

```diff
@@ -0,0 +1,26 @@
+use std::cell::RefCell;
+use std::iter::repeat_with;
+
+thread_local! {
+    static RNG: RefCell<fastrand::Rng> = RefCell::new(fastrand::Rng::new());
+}
+
+#[cxx::bridge(namespace = "kj::rust")]
+pub mod ffi {
+    extern "Rust" {
+        /// Fill a buffer with random bytes using a non-cryptographic PRNG.
+        /// The buffer can be any length.
+        fn fill_random_bytes(buffer: &mut [u8]);
+    }
+}
+
+/// Fills the provided buffer with random bytes using a fast non-cryptographic PRNG.
+/// Uses a thread-local Rng instance for efficiency and thread-safety.
+pub fn fill_random_bytes(buffer: &mut [u8]) {
+    RNG.with(|rng| {
+        let mut rng = rng.borrow_mut();
+        for (dest, src) in buffer.iter_mut().zip(repeat_with(|| rng.u8(..))) {
+            *dest = src;
+        }
```

Comment:
I assume it would be more efficient to use the "fill()" method provided by the library?

https://docs.rs/fastrand/latest/fastrand/fn.fill.html
https://github.com/smol-rs/fastrand/blob/master/src/lib.rs#L584

Reply:
FWIW, it looks like the library seeds with just the thread ID and current time...  I don't know exactly how those get measured... I assume they're probably OK, but it might be better to seed with a higher quality random value instead?